src/pickup.cpp

```diff
@@ -83,9 +83,13 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
     const bool can_be_folded = veh->is_foldable();
     const bool is_convertible = ( veh->tags.count( "convertible" ) > 0 );
     const bool remotely_controlled = g->remoteveh() == veh;
+    const int washing_machine_part = veh->part_with_feature( veh_root_part, "WASHING_MACHINE" );
+    const bool has_washmachine = washing_machine_part >= 0;
+    bool washing_machine_on = veh->parts[washing_machine_part].enabled;
```

Comment:
Segfault on this line, you cannot dereference an array with the -1 returned by veh->part_with_feature( veh_root_part, "WASHING_MACHINE" ); if there is no washing machine in the vehicle.
This would work:
```
bool washing_machine_on = ( washing_machine_part == -1 ) ? false :  veh->parts[washing_machine_part].enabled;
```

Reply:
I fixed this during the merge, looks like it works flawlessly otherwise :+1:

Reply:
Thanks!
Though I'd like to know at which moment does it segfault?